adhocracy4/images/mixins.py

```diff
@@ -0,0 +1,61 @@
+from django import forms
+from django.utils.functional import lazy
+from django.utils.translation import gettext_lazy as _
+
+
+# FIXME: backportet from django 1.11. remove this after update
+def _format_lazy(format_string, *args, **kwargs):
+    return format_string.format(*args, **kwargs)
+
+
+format_lazy = lazy(_format_lazy, str)
+
+
+class ImageMetadataMixin(forms.ModelForm):
+    def __init__(self, *args, image_name="image", **kwargs):
+        self.image_name = image_name
+        super().__init__(*args, **kwargs)
+        if self.instance.image:
+            self.initial["image_copyright"] = True
+            self.initial["image_alt_text"] = True
+
+    image_copyright = forms.CharField(
+        required=False,
+        label=format_lazy(_("{image_name} copyright"), image_name="image"),
+        max_length=120,
+        help_text=format_lazy(
+            _("Copyright shown in the {image_name}."), image_name="image"
+        ),
+    )
+
+    image_alt_text = forms.CharField(
+        required=False,
+        label=format_lazy(_("{image_name} alt text"), image_name="image"),
+        max_length=80,
+        help_text=format_lazy(
+            _(
+                "An alternate text serves as a textual description "
+                "of the image content and is read out by screen "
+                "readers. Describe the image in max. 80 characters. "
+                "Example: A busy square with people in summer."
+            ),
+            image_name="image",
+        ),
+    )
+
```

Comment:
I think you dont need to add/define the fields here, they should be there just from defining them in the model and then adding them to the form, so like [here](https://github.com/liqd/adhocracy4/blob/e864a4e2d505fa19c586f406f9b98944b0afe2a2/adhocracy4/dashboard/forms.py#L28), but I am not sure which the right form is here, you probably also have to add it manually to all the external, container etc forms.. 
So in the mixin only the __innit__ with the checks should be needed. And then you add the mixin to the ProjectCreateForms.

Reply:
I think I found the form, its in A4 [here](https://github.com/liqd/adhocracy4/blob/e864a4e2d505fa19c586f406f9b98944b0afe2a2/adhocracy4/dashboard/forms.py#L62). If we add it there, we also add it to all other projects, so we probably have to overwrite again in mB :frowning_face:

Reply:
@Rineee do you mean a+ ect.? I think as we want to have a default for all old images it's probably fine as we will want it in a+ for ally, although i think it's best practice to leave it blank if it's decorative but again it is not used like that so having alt is probably better then nothing